client/blocks/inline-help/popover.jsx

```diff
@@ -303,6 +369,8 @@ function mapStateToProps( state ) {
 		showOptIn: showSwitchEditorButton && optInEnabled && isCalypsoClassic,
 		gutenbergUrl,
 		isCheckout: section.name && section.name === 'checkout',
+		isAtomic,
+		sitePlugins,
```

Comment:
What about returning directly `classicPlugin: find( sitePlugins, { slug: 'classic-editor' } )`?
It would save us a couple of unnecessary repetitions.

Reply:
Nice, that is tidier - done!